docs/pipelines.md

```diff
@@ -54,10 +54,10 @@ A `Pipeline` definition supports the following fields:
         should execute after one or more other `Tasks` without output linking.
       - [`retries`](#using-the-retries-parameter) - Specifies the number of times to retry the
         execution of a `Task` after a failure. Does not apply to execution cancellations.
-      - [`conditions`](#specifying-conditions) - Specifies `Conditions` that only allow a `Task`
+      - [`conditions`](#specifying-execution-conditions) - Specifies `Conditions` that only allow a `Task`
         to execute if they evaluate to `true`.
       - [`timeout`](#configuring-the-failure-timeout) - Specifies the timeout before a `Task` fails. 
-  - [`results`](#pipeline-results) - Specifies the file to which the `Pipeline` writes its execution results.
+  - [`results`](#monitoring-execution-results) - Specifies the file to which the `Pipeline` writes its execution results.
```

Comment:
this is taking us to task level execution results at https://github.com/tektoncd/pipeline/blob/master/docs/pipelines.md#monitoring-execution-results instead of Pipeline Results at https://github.com/tektoncd/pipeline/blob/master/docs/pipelines.md#monitoring-execution-results-1

Please update it to `monitoring-execution-results-1` instead of `monitoring-execution-results`

Reply:
Thanks, you are right, fixed